bin/pytorch_inference/CCommandParser.cc

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+#include "CCommandParser.h"
+
+#include <core/CLogger.h>
+
+#include <iostream>
+
+#include <rapidjson/error/en.h>
+#include <rapidjson/istreamwrapper.h>
+#include <rapidjson/stringbuffer.h>
+#include <rapidjson/writer.h>
+
+namespace ml {
+namespace torch {
+
+namespace {
+void debug(const rapidjson::Document& doc) {
+    rapidjson::StringBuffer buffer;
+    rapidjson::Writer<rapidjson::StringBuffer> writer(buffer);
+    doc.Accept(writer);
+    LOG_TRACE(<< buffer.GetString());
+}
```

Comment:
I wonder if we should move this to `core/CRapidJsonUtils.h`. This pattern comes up a certain amount and it would be good to have a single definition for this.

Reply:
I added `operator<<` to this file. It is a trivial function that has to live in the `rapidjson` namespace so I don't think it belongs in `core/CRapidJsonUtils.h`

Reply:
It's this stuff I meant:
```
rapidjson::StringBuffer buffer;
rapidjson::Writer<rapidjson::StringBuffer> writer(buffer);
doc.Accept(writer);
```
i.e. conversion to string which I'm sure I've put somewhere local myself in the past. I find I have to remind myself every time how to convert a `rapidjson::Document` to a string and thought perhaps it was time to make this a utility somewhere. That said you don't have to make this change and perhaps we should hunt for other cases address them all in one go.